src/transaction/declare_v2.rs

```diff
@@ -190,6 +215,10 @@ impl DeclareV2 {
         Ok(())
     }
 
+    /// Execute the contract in the cairo-vm. Returns a TransactionExecutionContext if succesful.
+    /// ## Parameter:
+    /// - resources: the resources that are in use by the contract
+    /// - block_context: The block that contains the execution context
```

Comment:
this is not correct

Reply:
i change the description but which would the incorrect part?